src/components/CreateShareStructure/ListShareClass.vue

```diff
@@ -362,7 +362,7 @@ export default class ListShareClass extends Vue {
   }
 
   .actions-dropdown_item {
-    min-height: 0!important;
+    min-height: 0 !important;
     margin: 1rem 0;
   }
 }
```

Comment:
I've made a new component that will be an amalgamation of the two previous ShareClass Comps, so i will make sure i implement these changes there!